src/models/v3/security-requirements.ts

```diff
@@ -0,0 +1,16 @@
+import { Collection } from '../collection';
+
+import type { SecurityRequirementsInterface } from '../security-requirements';
+import type { SecurityRequirementInterface } from '../security-requirement';
+
+export class SecurityRequirements extends Collection<SecurityRequirementInterface> implements SecurityRequirementsInterface {
+  override get(id: string): SecurityRequirementInterface | undefined {
+    // return this.collections.find(trait => trait.id() === id);
```

Comment:
Is this a TODO?

Reply:
Yes, and no. Because of the fact that these objects do not have `id` I wonder if we should return `undefined`, or change this collection to a regular array and not collection class.

Reply:
Good point. Can't we do just directly use `Record<string, { schema: SecuritySchemeInterface, scopes: Array<string> }>` ?

Reply:
We cannot, `security` field (in security and operation) is an array, so we ca change it to the 

```ts
Array<Record<string, { schema: SecuritySchemeInterface, scopes: Array<string> }>>
```

Reply:
I think makes sense